benchmarks/README.md

```diff
@@ -0,0 +1,112 @@
+# Mesa Performance Benchmarks
+
+This directory contains tools for benchmarking Mesa model performance. These benchmarks help track performance improvements and regressions across different versions of Mesa.
+
+## Available Files
+
+- `configurations.py`: Contains model configurations for benchmarking
+- `global_benchmark.py`: Main script for running benchmarks
+- `compare_timings.py`: Tool to compare results between benchmark runs
+
+## How to Use
+
+
+### 1. Benchmark Configuration
+
+The `configurations.py` file defines which models to benchmark and their parameters. Each model has:
+
+- `small` and `large` configurations for testing different scales
+- Parameters for:
+  - `seeds`: Number of different random seeds to use
+  - `replications`: Number of times to repeat the benchmark for each seed
+  - `steps`: Number of model steps to run
+  - `parameters`: Model-specific parameters
+
+Example of adding a new model to benchmark:
```

Comment:
I would advocate against including this. When we added the benchmarks, we agreed on the models to use for this. So I would only explain in the readme how to run the existing benchmarks, rather than how to add new models to the benchmarks.